uniprot-config/src/test/java/org/uniprot/store/config/returnfield/config/impl/UniParcReturnFieldConfigImplIT.java

```diff
@@ -117,7 +109,9 @@ private static Stream<Arguments> provideReturnSortFields() {
 
     private static Stream<Arguments> provideReturnFieldNames() {
         return returnFieldConfig.getReturnFields().stream()
+                .filter(rf -> Objects.nonNull(rf.getChildNumber()))
                 .map(ReturnField::getName)
+                .filter(name-> !name.equals("common_taxons"))
```

Comment:
We should not filter common_taxons here. If it is failing with it, it means there is an issue. We also have to implement in for TSV, maybe this is the issue.

Reply:
Supun has added config test based on UniParcLightEntry